mixin-utils/utils.libsonnet

```diff
@@ -1,10 +1,12 @@
 local g = import 'grafana-builder/grafana.libsonnet';
 
 {
-  // The classicNativeHistogramQuantile function is used to calculate histogram quantiles from native histograms or classic histograms.
-  // Metric name should be provided without _bucket suffix.
-  // If from_recording is true, the function will assume :sum_rate metric suffix and no rate needed.
-  nativeClassicHistogramQuantile(percentile, metric, selector, sum_by=[], rate_interval='$__rate_interval', multiplier='', from_recording=false)::
+  // The ncHistogramQuantile (native classic histogram quantile) function is
+  // used to calculate histogram quantiles from native histograms or classic
+  // histograms. Metric name should be provided without _bucket suffix.
+  // If from_recording is true, the function will assume :sum_rate metric
+  // suffix and no rate needed.
+  ncHistogramQuantile(percentile, metric, selector, sum_by=[], rate_interval='$__rate_interval', multiplier='', from_recording=false)::
```

Comment:
If it supports both native an classic, why do we mention `nc` at all? Can it just be `histogramQuantile`?

Reply:
that doesn't imply input/output of the `{classic: '...', native: '...'}` form and will make it harder to search for stuff

Reply:
also it's for me to keep track of native histograms migration and keep my sanity